pkgs/applications/version-management/git-and-tools/gitstatus/romkatv_libgit2.nix

```diff
@@ -2,13 +2,15 @@
 
 libgit2.overrideAttrs (oldAttrs: {
   cmakeFlags = oldAttrs.cmakeFlags ++ [
+    "-DBUILD_CLAR=OFF"
+    "-DBUILD_SHARED_LIBS=OFF"
+    "-DREGEX_BACKEND=builtin"
     "-DUSE_BUNDLED_ZLIB=ON"
+    "-DUSE_HTTPS=OFF"
+    "-DUSE_HTTP_PARSER=builtin"
```

Comment:
Can you add a note to the keys your overriding from the main derivation?

Reply:
Done!